### DISCUSSION TITLE
New tab keyboard shortcut

### DISCUSSION THREAD

**Discussion by MANICX100** [Eugeny/tabby#6257] [Q&A] [answered]
I'm pretty sure I am being dense but what is the keyboard shortcut to create a new tab?

Can this be edited via config file

**Comment by Eugeny** [comment, marked as answer]
Ctrl-Shift-T or Cmd-T - you can go to Settings -> Hotkeys and search for `new tab` to change it

**Reply by MANICX100** [reply]
simple thank you

**Reply by lawweiliang** [reply]
@MANICX100 

Just to share my 2 cent, I standardize all my hotkey into 
Ctrl+Shift + [x] -> Tab
Ctrl+Alt + [x] -> Pane

---
Tab
![image](https://user-images.githubusercontent.com/34936929/228130442-9a91823d-30fd-493e-b000-96eeacbeaae1.png)

---
Pane
![image](https://user-images.githubusercontent.com/34936929/228130495-3d5f80c9-ff21-4376-888e-e1db1dce266d.png)

---

**Comment by lawweiliang** [comment]
@MANICX100  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
